src/containers/Nav/NavNotificationBar.tsx

```diff
@@ -14,6 +12,21 @@ const throwMockError = () => {
   }
 }
 
+export const FeedBackButton = () => {
+  return (
+    <Button
+      color="primary"
+      onClick={throwMockError}
+      size="small"
+      style={{
+        textTransform: 'none',
+      }}
+    >
+      {texts.nav.notificationBar.feedback}
+    </Button>
```

Comment:
Maybe prefer this writing : 

```
import { Button as ButtonBase } from 'src/components/Button'

const Button = styled(ButtonBase)`
  && {
   text-transform: none
 }
 `
```

Reply:
Mmmmh j'avais fais ça sur ma première PR mais @GuillaumeJasmin m'avait dit que ce n'était pas la bonne manière de surchargé les attributs des composants de cette manière.

Reply:
Mmmmh j'avais fais ça sur ma première PR mais @GuillaumeJasmin m'avait dit que ce n'était pas la bonne manière de surchargé les attributs des composants de cette manière. A part si je confonds ?

Reply:
https://github.com/ReseauEntourage/entourage-webapp/pull/32#discussion_r560518608